packages/core/bootstrap/src/index.ts

```diff
@@ -195,7 +221,8 @@ export const executeSync: ExecuteSync = async (
 
     return callback(result.statusCode, result)
   } catch (error) {
-    return callback(error.statusCode || 500, Requester.errored(data.id, error))
+    const feedID = metrics.util.getFeedId(data)
+    return callback(error.statusCode || 500, Requester.errored(data.id, error, undefined, feedID))
```

Comment:
The third argument into `Requester.errored` should be `error.statusCode`.

Reply:
resolved in #1058